configure.ac

```diff
@@ -142,25 +142,26 @@ AS_IF([test "x$have_jsoncpp" = "xyes"], [
 ])
 
 AC_ARG_WITH([openssl], AS_HELP_STRING([--with-openssl], [Build with OpenSSL support]))
-AS_IF([test "x$with_openssl" != "xno"],
+AS_IF([test "x$with_openssl" = "xyes"],
```

Comment:
Should default to use openssl except if --without-openssl is passed.
This would build with openssl only if --with-openssl is used

Reply:
Done at ad695035aa25ff725c51447a3aa5fb5241e9cfe2